python/temba_expressions/tests.py

```diff
@@ -737,6 +737,9 @@ def test_tokenize(self):
         self.assertEqual(tokenize("this is a sentence"), ["this", "is", "a", "sentence"])
         self.assertEqual(tokenize("  hey  \t@ there  "), ["hey", "there"])
         self.assertEqual(tokenize("واحد اثنين ثلاثة"), ["واحد", "اثنين", "ثلاثة"])
+        self.assertEqual(tokenize("we win @game and we \U0001F64C"), ["we", "win", "game", "and", "we", "\U0001F64C"])
+        self.assertEqual(tokenize(""), [])
+        self.assertEqual(tokenize("\n @ ."), [])
```

Comment:
What happens if you multiple emojis in a row? Would be good to have tests there as well. (I would expect them to be separate tokens I think)